ddp-workspace/projects/ddp-angio/src/app/app.module.ts

```diff
@@ -134,7 +134,7 @@ export function translateFactory(translate: TranslateService, injector: Injector
 export class AppModule {
   constructor(
     private router: Router,
-    private analytics: GoogleAnalyticsEventsService) {
+    private analytics: AnalyticsEventsService) {
     this.router.events.subscribe(event => {
       if (event instanceof NavigationEnd) {
         this.analytics.emitNavigationEvent();
```

Comment:
Is it possible to factor this into toolkit? Guess we would need access to router?

Reply:
Ok, I’ll try and see how it will work in the Toolkit.